File: test/aidbox_sdk/snapshots/typescripthl7-fhir-r4-core/AppointmentResponse.ts
import { CodeableConcept } from "./CodeableConcept";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";

export type AppointmentResponse = DomainResource & {
    end?: string;
    actor?: Reference;
    start?: string;
    comment?: string;
    identifier?: Identifier[];
    appointment: Reference;
    participantType?: CodeableConcept[];
    participantStatus: string;
};

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/AppointmentResponse.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import CodeableConcept
from base import DomainResource
from base import Reference
from base import Identifier

@dataclass(kw_only=True)
class AppointmentResponse(DomainResource):
    end: Optional[str] = None
    actor: Optional[Reference] = None
    start: Optional[str] = None
    comment: Optional[str] = None
    identifier: Optional[List[Identifier]] = field(default_factory=list)
    appointment: Reference
    participant_type: Optional[List[CodeableConcept]] = field(default_factory=list)
    participant_status: str